Source: package maintainer
Upstream: no
Reason: There's no "lp" user on Void Linux

--- a/opt/brother/Printers/dcpl8410cdw/inf/setupPrintcapij
+++ b/opt/brother/Printers/dcpl8410cdw/inf/setupPrintcapij
@@ -34,9 +34,8 @@ case "$2" in
 	if [ ! -d $SPOOLER_NAME ]; then
 	        mkdir -p $SPOOLER_NAME
 	fi
-	chown lp $SPOOLER_NAME
 	chgrp lp $SPOOLER_NAME
-	chmod 700 $SPOOLER_NAME
+	chmod 770 $SPOOLER_NAME

 	if [ "$3" = "USB" ]; then
 		DEVICE_IF=/dev/usb/lp0
